client/components/tinymce/plugins/advanced/style.scss

```diff
@@ -20,6 +20,7 @@
 }
 
 .mce-toolbar .mce-btn-group .mce-advanced.mce-btn.mce-last {
+	background-color: rgba( 0, 0, 0, 0 );
```

Comment:
Thanks for looking into this issue! Let's use the sass variable `$transparent` which is defined in `_colors.scss` We try to avoid using custom colors when we can. See: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#the-dry-principle